### ISSUE TITLE
Please add armv7ve build profile

### ISSUE THREAD

**Issue by falkTX** [sadko4u/lsp-plugins#57]
Raspberry Pi3 is the most common ARM board, its native "-march" parameter is `armv7ve`.
So I think it makes sense to have a build profile for this.
I am building packages for it right now, but get this build warning:
```
cc1plus: warning: switch -mcpu=cortex-a7 conflicts with -march=armv7-a switch
```

**Comment by sadko4u** [comment]
Hmm, the official doc says:
```
‘armv7ve’
    The extended version of the ARMv7-A architecture with support for virtualization. 
```
So, it's an advanced armv7-a.
Maybe there's something other with ```-mcpu=cortex-a7``` switch...

**Comment by falkTX** [comment]
armv7ve is the architecture of the cortex-a7 cpu (it has those virtualization extras)

is there a way to build without applying a "build profile" and let the one compiling the code supply all optimizations flags?
if so, then I can use that instead of trying to match architectures.

**Comment by sadko4u** [comment]
I'm currently working on it.
I'll also add 'arm32' profile that will not set ```-march``` settings and will use system defaults instead.

**Comment by sadko4u** [comment]
Sorry for a delay. Because by default Raspbian is building binaries for ARMv6 architecture, was required to implement some support for this architecture. Please test the changes committed to the branch **github-issue-57**.
Added two profiles:
* armv7ve - for building for ARMv7VE architecture.
* arm32 - for building for ARM with default OS compiler settings.

**Comment by sadko4u** [comment]
@falkTX how it's going?

**Comment by falkTX** [comment]
I have a test build running here https://launchpad.net/~kxstudio-debian/+archive/ubuntu/staging/+sourcepub/10568174/+listing-archive-extra
Seems to work well, though built has not finished as I write this comment.

**Comment by sadko4u** [comment]
If all will be OK, I'll do a hotfix patch to the master branch.

**Comment by falkTX** [comment]
Everything builds fine. No warnings on the armhf build

**Comment by sadko4u** [comment]
Merged changes to the **devel** branch: 6f0df4108da9684a027ddce34f0120c48bdd56b0
Applied patch to the **hotfix** branch: 76b0604b81759a0b08f9974f2323b8246aa23701
Removed **github-issue-57** branch.

**Comment by sadko4u** [comment]
Available in the 1.1.11 release!
